### DISCUSSION TITLE
quality of Dutch translations

### DISCUSSION THREAD

**Discussion by axd1967** [Stellarium/stellarium#1999] [Translations]
I am part of the Dutch translation team but noticed that a some translations are not quite correct:

- "Sea of Waves" is translated to "Zee van golfen" while it should probably be "Zee van golven"
- "Sea of Crises" is translated as "Zee van crises" but should be "Zee van gevaren" (https://nl.wikipedia.org/wiki/Mare_Crisium)
- "Southwestern crater province" was not translated at all, but copied as such; many more strings appear to have been copied without translation
- "Far-side southern lowland" has quite an odd translation "Ver zuidelijke laagland"
- "Armintxe Cave" becomes "Armintxe grot" but "Grot van Armintxe" is better

**Comment by gzotti** [comment]
If you have a better translation, please propose it at Transifex,exclusively. Here is not the place to discuss these. Who if not the Dutch translators should solve translation issues?

**Reply by axd1967** [reply]
The issue I'm raising here is that the Dutch translation quality is not very good. In other words, the translator didn't do his/her best _at all_ and the result makes me question his/her mastery of the Dutch language. Maybe a (mediocre) tool was used in the translation?

The few observations I'm listing above should be an indication that _that_ Dutch translator is maybe not the best translator. Any Dutch native speaker will recognise that those translations are of a questionable quality.

If there is no good translation available, better leave it untranslated?  If all strings are badly translated (maybe to get rid of the problem - or rather, swipe it under the carpet), the work remains to be done after all, because someone else still has to do the job, so better leave it untranslated? The recent introduction of a lot of names has made the job of translators quite heavy (see #1225, )

As far as I can see, there is no dedicated forum on Transifex to discuss this, but I don't want to put names here.

**Reply by alex-w** [reply]
The Dutch translation team has 14 members and I see that these members are work except you. If you see mistake in translation or you may propose the better translation then... do it! Please do not propose remove all translations anymore (probably with including Dutch language) if you do not like some translations. 

Or, at least if all translation team want it, we may remove Dutch language with all translations.

**Reply by axd1967** [reply]
This is again an extremely rude reaction. Who on this world would ever propose to remove all Dutch translations??? What kind of discussion is this? And why do you say that I do not work as translator ("_these members are work except [for] you_")? 

I will organise a chat with all these Dutch translators.

**Reply by alex-w** [reply]
> This is again an extremely rude reaction. 

Because some of your proposals are very strange and they are shocked me.

> Who on this world would ever propose to remove all Dutch translations??? 

Let me quote your proposal:
> If there is no good translation available, better leave it untranslated? If all strings are badly translated (maybe to get rid of the problem - or rather, swipe it under the carpet), the work remains to be done after all, because someone else still has to do the job, so better leave it untranslated?

You are coordinator of Dutch translation team and you have tool to review the translations and approve good one or fix/improve the exist bad translations. Where is your fixes/reviews? The policy "better leave it untranslated" is good for languages, which doesn't have special terms, but for Dutch language this will be a shame. 

> What kind of discussion is this? 

Because criticise just to criticise is counterproductive. You may publish announce to all Dutch translation team on [Transifex](https://www.transifex.com/stellarium/teams/80998/discussions/) or here, but please be productive - please review the translations, add special terms in the glossary and work together to all members of translation team. 

> And why do you say that I do not work as translator ("_these members are work except [for] you_")?

Because I looked at log of changes in translations in last month. But yes, you are translated few phrases a one month and one week ago - sorry, I was not fully correct and you are translate the GUI and other stuff too.

> I will organise a chat with all these Dutch translators.

Go ahead and good luck!

**Reply by axd1967** [reply]
> The Dutch translation team has 14 members and I see that these members are work except you. 

As far as I can see, of those 14, most are no longer active. In fact, only a handful have reacted to me contacting them, and only a very few of those are really interested in going forward. This is a rather disconcerting state.

And I think I have done a decent part of the translations as well as reviewing/correcting exiting ones, so please stop suggesting me of **not working at all**. THAT is extremely rude.

**Comment by Kogacarlo** [comment]
I agree that some translations are not the best. Usually when I see this, for example while using Stellarium, I dive into Transifex to correct it.
By the way my Transifex name is Pawnstar.

**Reply by alex-w** [reply]
Thank you very much for your work!

**Comment by axd1967** [comment]
Dutch participants for now

- :heavy_check_mark: axd1967
- :heavy_check_mark: eddeig 
- :x: pahles 
- :x:  HermMensch 
- Pawnstar 
- :heavy_check_mark: Dronepilot 
- :new:  :x:  Yodo


upcoming meeting 

- date/time 2021-11-25 2100-2200(+01:00)
- via private channel, Jitsi; participants have been informed

Topics to discuss

1. round table
2. short Jitsi presentation
1. issues with the Transifex user interface
1. problems with source terms, if any (basically not a translator issue, but maybe we can open issues and/or propose recommendations to the Stellarium team, and/or extend the discussion to include other language teams)
1. how to deal with difficult translations (https://github.com/Stellarium/stellarium/issues/1225 contains valuable suggestions)
2. how to deal with incorrect translations
1. how to deal with the wave of new terms (again #1225)
2. how to communicate intra-team
1. remarks - if any - on existing translations
1. AOB - any other business

Part of these subjects could be of interest to Stellarium team as well as other translator teams, this can always be organised separately should the need arise; the main idea is to get together and have an informal chat. Outcome is open for the public and I will append a small report (in English) to this conversation.

(Dutch translators: feel free to suggest more topics, I will add those to the list above.)

**Reply by alex-w** [reply]
I hope you not will propose increasing the number of translatable string via adding context data or split on hundernd of resources just because you ignore documentation for filtering at Transifex.

**Comment by axd1967** [comment]
@Kogacarlo I haven't received your reaction on the invitation. Can you please check your messages on Transifex?

**Comment by axd1967** [comment]
meeting notes

- keep latin names as such (add between brackets? -> avoid, as Transifex will balk on this)
- leave untranslated rather than copy terms that might one day be translated
- many translations should not have been translated (eg chinese) ; they have no use because they cannot be found back in translated form; do not attempt to translate without understanding the context => solution: review + remove unneeded translations 
- there is no need to translate "everything"!
- we should talk to each other from time to time. this should not be often, but from time to time; eg release date + 2 months (and when needed etc)
- inactive users removed: DVR, GerardP, Yoriyari, cardinot, jeroenbeuz, jorrit02, kvwyns (if interested, these people can always rejoin)
- is it possible to put chinese terms in a separate "collection" so that the overall amount of translation is fractioned

**Comment by axd1967** [comment]
Related: https://github.com/Stellarium/stellarium/issues/2738
